matrix-meetings-bot/src/shared/calendarUtils/extractCalendarChange.test.ts

```diff
@@ -0,0 +1,378 @@
+/*
+ * Copyright 2023 Nordeck IT + Consulting GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { mockCalendarEntry } from '../../testUtils';
+import { extractCalendarChange } from './extractCalendarChange';
+
+describe('extractCalendarChange', () => {
+  it.each([
+    {
+      name: 'update existing single entry',
```

Comment:
I didn't change that, it feels a bit more fine to me when that these tests are grouped into the some behavior. But these list is smaller now because time and rrule changes are moved to separate tests.